ucm2/USB-Audio/USB-Audio.conf

```diff
@@ -67,13 +67,22 @@ If.steinberg-ur44 {
 
 If.M4 {
 	Condition {
-		Type String
-		Haystack "${CardComponents}"
-		Needle "USB07fd:000b"
+		Type RegexMatch
```

Comment:
This is very bad condition. It will match any USB cards with `M4` in the `CardName`. Keep the `USB07fd:000b` check and extend the `True` block like:

```
If.Motu {
  Condition {
    Type String
    Haystack "${CardComponents}"
    Needle "USB07fd:000b"
  }
  True.If.M4 {
    Condition {
      Type String
      Haystack "${CardLongName}"
      Needle "MOTU M4"
    }
    True.Define.ProfileName "MOTU/M4"
    False.Define.ProfileName "MOTU/M2"
  }
}
```